src/api/user/login.js

```diff
@@ -21,10 +21,10 @@ export const checkMobile = async (mobile) => {
   }
 };
 
-export const login = async (body) => {
+export const login = async (mobile) => {
   try {
     const reqUrl = `${backendApiUrl}/user/auth/login`;
-    const result = await axios.post(reqUrl, body);
+    const result = await axios.post(reqUrl, mobile);
```

Comment:
this is not the correct syntax. you can't send mobile directly in the body , the body must be a object check postman for reference